packages/mermaid/src/Diagram.ts

```diff
@@ -95,30 +68,20 @@ export class Diagram {
   }
 }
 
-export const getDiagramFromText = (
-  txt: string,
-  parseError?: ParseErrorFunction
-): Diagram | Promise<Diagram> => {
+export const getDiagramFromText = async (txt: string): Promise<Diagram> => {
```

Comment:
Maybe change name of argument txt->text as done in row 16

Reply:
Done